data/help.txt

```diff
@@ -50,6 +50,7 @@ help "outfitter"
 	`Some types of outfits have other requirements as well. For example, only some of your outfit space can be used for engines or weapons; this is your ship's "engine capacity" and "weapon capacity." Guns and missile launchers also require a free "gun port," and turrets require a free "turret mount." Also, missiles can only be bought if you have the right launcher installed.`
 	`Use your scroll wheel, or click and drag, to scroll the view.`
 	`As in the trading panel, you can hold down Shift or Control to buy 5 or 20 of an outfit at once, or both keys to buy 100.`
+	`If you want to buy an outfit into your cargo hold, you can hold down Control while clicking any selected ships to deselect them.`
```

Comment:
I wanted to match the previous sentence, hence the `you can hold` and `to deselect`, i.e.  a more 'informational' than 'instructional' tone.